testsuite/src/it/java/io/quarkus/ts/startstop/CodeQuarkusTest.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2020 Contributors to the Quarkus StartStop project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package io.quarkus.ts.startstop;
+
+import io.quarkus.ts.startstop.utils.Apps;
+import io.quarkus.ts.startstop.utils.CodeQuarkusExtensions;
+import io.quarkus.ts.startstop.utils.MvnCmds;
+import io.quarkus.ts.startstop.utils.URLContent;
+import io.quarkus.ts.startstop.utils.WebpageTester;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.logging.Logger;
+
+import static io.quarkus.ts.startstop.utils.Commands.cleanDir;
+import static io.quarkus.ts.startstop.utils.Commands.download;
+import static io.quarkus.ts.startstop.utils.Commands.getArtifactGeneBaseDir;
+import static io.quarkus.ts.startstop.utils.Commands.getBuildCommand;
+import static io.quarkus.ts.startstop.utils.Commands.parsePort;
+import static io.quarkus.ts.startstop.utils.Commands.processStopper;
+import static io.quarkus.ts.startstop.utils.Commands.runCommand;
+import static io.quarkus.ts.startstop.utils.Commands.unzip;
+import static io.quarkus.ts.startstop.utils.Commands.waitForTcpClosed;
+import static io.quarkus.ts.startstop.utils.Logs.archiveLog;
+import static io.quarkus.ts.startstop.utils.Logs.checkLog;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * @author Michal Karm Babacek <karm@redhat.com>
+ */
+@Tag("codequarkus")
+public class CodeQuarkusTest {
```

Comment:
Tests pass with OpenJDK 11, but fail using GraalVM CE 19.3.1 with unexpected message `OpenJDK 64-Bit Server VM warning: forcing TieredStopAtLevel to full optimization because JVMCI is enabled`

This message should be whitelisted

```
[ERROR] Failures:
[ERROR]   CodeQuarkusTest.mixExtensions:135->testRuntime:97 MVNW_DEV log should not contain error or warning lines that are not whitelisted. See testsuite/target/archived-logs/io.quarkus.ts.startstop.CodeQuarkusTest/mixExtensions/dev-run.log and check these offending lines:
OpenJDK 64-Bit Server VM warning: forcing TieredStopAtLevel to full optimization because JVMCI is enabled ==> expected: <true> but was: <false>
```